src/Sulu/Bundle/AdminBundle/Tests/Features/change_language.feature

```diff
@@ -8,5 +8,12 @@ Feature: Change the language of the backoffice
 
     Scenario: Change the language
         Given I am on "/admin"
+        And I wait a second
```

Comment:
Yes this is necessary, because in the previous point a page reload (from the login to the admin) happens. If one would - in the second step - try to touch anything remotely connected to Husky or Sulu a nasty "undefined" error is thrown.